packages/nextjs/tailwind.config.js

```diff
@@ -71,6 +73,15 @@ module.exports = {
         "space-grotesk": ["Space Grotesk", "sans-serif"],
         "neue-machina": ["PPNeueMachina", "sans-serif"],
       },
+      backgroundColor: {
+      'base-400': '#EBECFD',
+      },
+      boxShadow: {
+        'custom': '0px 0px 11.65074px 0px rgba(139, 178, 241, 0.50)',
```

Comment:
NITPICK for names : 

`base-400` -> `skin` ? 

'custom' -> `surround` (I think the reason why we added this because tailwind shadow classes has more shadow at bottom then surrounding right ? ) or maybe use `even` so that while using  it will be `shadow-even` ?

Reply:
Names updated to `skin` and `even`! 👍🙌